### PR TITLE
CASMINST-5009

### PR DESCRIPTION
**Pull request by cdelatte-hpe**
### Summary and Scope

* Fix for podman registry lookup syntax.
* added python virtual env for use with the csm rbd mapper tool

- Fixes: CASMINST-5009 & CASMPET-5851

#### Issue Type

- Bugfix Pull Request

### Prerequisites

- [ ] I have included documentation in my PR (or it is not required)
- [ ] I tested this on internal system (if yes, please include results or a description of the test)
- [ ] I tested this on a vshasta system (if yes, please include results or a description of the test)

### CODE REVIEW

**Review by heemstra — APPROVED** [review]

File: boxes/ncn-common/provisioners/common/setup.sh [hunk 1]
```diff
@@ -55,6 +55,7 @@ python3 -m venv /etc/ansible/csm_ansible
 pip3 install --upgrade pip
 pip3 install ansible-core==$ANSIBLE_VERSION
 pip3 install ansible
+pip3 install fabric
```
Comment (rustydb) [inline boxes/ncn-common/provisioners/common/setup.sh]: Just add `fabric` to the `REQUIREMENTS` array above.

Comment (rustydb) [reply boxes/ncn-common/provisioners/common/setup.sh]: Wait this is confusing, looking at another commit this fabric one is for a different env.?

Comment (rustydb) [reply boxes/ncn-common/provisioners/common/setup.sh]: Looking at the last commit I think I answered my own questions: https://github.com/Cray-HPE/node-images/pull/423/commits/dba99ddf8daaaf3f433f362a169d3c77a2d7fb2f

Comment (cdelatte-hpe) [reply boxes/ncn-common/provisioners/common/setup.sh]: yeah, sorry just saw this.  it is for a diff venv

**Pull request merged**